components/MobileNavbar.tsx

```diff
@@ -10,17 +10,17 @@ export const MobileNavBar = () => {
         <footer className="fixed bottom-1 w-1/11  left-0 right-0 bg-gray-800 text-white py-4 md:hidden rounded-xl mx-5">
             <div className="container mx-auto text-xl flex justify-between items-center px-4">
                 <Link href="/">
-                    <AiFillHome />
+                    <AiFillHome className="text-primary hover:text-primary-dark"/>
```

Comment:
hey @MeenuyD add only hover effect to `primary` not normal text color.

now just remove the classnamme 
```suggestion
                    <AiFillHome className="hover:text-primary"/>
```